File: timeConversion.js
function timeConversion(s) {
    const timeSplit = s.split(':')
    let formattedTime = ""
    if(timeSplit[timeSplit.length - 1].endsWith('PM') && Number(timeSplit[0]) === 12) {
        timeSplit[0] = 12}
    else if(timeSplit[timeSplit.length - 1].endsWith('PM') && Number(timeSplit[0]) >= 1) {
        timeSplit[0] = Number(timeSplit[0]) + 12
    } else if(timeSplit[timeSplit.length - 1].endsWith('AM') && Number(timeSplit[0]) === 12) {
        timeSplit[0] = '00'
    }
   formattedTime = timeSplit.join(":")
   return formattedTime.substring(0, formattedTime.length - 2)
}

File: time_conversion.py
def timeConversion(s):
    if s[-2:] == 'PM' and int(s[0:2]) != 12:
        return str(int(s[0:2]) + 12) + s[2:-2]
    if s[:2] == '12' and s[-2:] == 'AM':
        return '00' + s[2:-2]
    else:
        return s[:-2]
